app/src/main/res/layout/fragment_transition_detail.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <LinearLayout
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/punchdrunker/hocho/blob/80bf390736e73024864d5df1290864b927e15134/app/src/main/res/layout/fragment_transition_detail.xml#L4"></span>This <code>LinearLayout</code> view is useless (no children, no <code>background</code>, no <code>id</code>, no <code>style</code>)</td>
    </tr>
  </tbody>
</table>